### DISCUSSION TITLE
Show full week at a time

### DISCUSSION THREAD

**Discussion by MelissaBo13** [iSoron/uhabits#2024] [Feature Requests]
It would be great if you could see the seven-day week at a glance, without having to turn your phone to landscape (mine is locked to portrait, so I have to unlock it before I can do that). But even in landscape, it shows a lot more than 7 days. It would much easier to figure out at a glance if you completed your habit for the week if it just showed seven days exactly. May I suggest doing this in portrait mode, by having the habit name on one line and the checkmarks on the next? Much like the Way of Life app.

Also, if the description could also show the desired frequency, that would be easier than having to click into each habit to see the frequency. 

Thank you for listening! I hope you'll consider these suggestions, because otherwise, it's a really great app!

**Comment by injainja** [comment]
+1

**Comment by KyleSCraig** [comment]
Can't you tell just from looking at the current day? Solid or hollow tick means you did it in the last week if that's your frequency. You can also do hide completed so you don't see the ones you have done.

**Reply by MelissaBo13** [reply]
Yes but if you're a visual person, you might want to see at a glance what days you did it in a week and which ones you didn't. Besides, the empty checkmarks are glitchy. When I set 1x per week, it automatically changes it to every 7 days. So if I complete it on a Wednesday, it shows empty checkmarks until the following Tuesday. But my week starts on Monday, so I'd like it to be clear starting Monday so I know I haven't completed it in that calendar week yet. I posted that as a separate issue.

Here's an example of what showing all 7 days looks like, and works for my brain much better.
![Screenshot_20240722_153907_Chrome](https://github.com/user-attachments/assets/6fd94cb1-e57a-46ed-8cc8-805879129961)

**Comment by hockeymikey** [comment]
This is a must for me. I view everything in terms of the week anf should be set at that. Ideally have a setting for people to configure the landscape vs vertical I think would be best as some might prefer less or be on a smaller phone.

**Comment by ArmandFrvr** [comment]
This is the one issue that's critical to me and probably going to dissuade me from using this app.  Looks really good otherwise but I need an at a glance full week view.
